src/Window.cpp

```diff
@@ -246,7 +246,16 @@ int Window::GetWidth() const
 
     const Wayland::Monitor* mon = Wayland::FindMonitorByName(m_MonitorName);
     ASSERT(mon, "Window: Couldn't find monitor");
-    return mon->width / mon->scale;
+
+    int viewed_width = mon->width;
```

Comment:
One last nitpick (I forgot to add it yesterday, sorry): Please use pascalCase (i.e. ```viewedWidth```)

Reply:
Should be good now